after/syntax/sml.vim

```diff
@@ -13,60 +13,69 @@
 "
 "       au! FileType sml setlocal conceallevel=2
 "
-
 " Fix wonky highlighting for => operator
 syn match smlOperator  "=>"
 
 " Also highlight 'NOTE' and 'Note' in comments
 syn keyword  smlTodo contained NOTE Note
 
-" Highlight type variables (i.e., tokens that look like 'xyz)
-syn match smlType "\<'\w\+\>"
```

Comment:
This is not "unused". It highlights the `'xyz` in snippets like this:

```sml
fun foo (x : 'xyz) : 'xyz = x
```

I tried looking into why adding this match fails on your patch. If you can't figure out why it doesn't work, I'd probably accept a patch that hides this behind `g:sml_greek_tyvar_show_tick` with a comment saying that it doesn't work otherwise.